modules/apps/frontend-js/frontend-js-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -34,7 +34,7 @@
 
 		<button id="test-button-react">Increment react</button>
 
-		<aui:script require="@liferay/frontend-js-state-web@1.0.3/index as StateModule, @liferay/frontend-js-sample-web@1.0.0/js/sharedState as SharedState">
+		<aui:script require="@liferay/frontend-js-state-web@1.0.6/index as StateModule, @liferay/frontend-js-sample-web@1.0.0/js/sharedState as SharedState">
```

Comment:
Hey John, about `@liferay/frontend-js-state-web@1.0.6`, did you already tried to use `@liferay/frontend-js-state-web`?

Reply:
Testing that out right now, your suggestion makes me feel better about my guess on the syntax :)

Reply:
It didn't work... :(

Using 
```
		<aui:script require="@liferay/frontend-js-state-web/index as StateModule, @liferay/frontend-js-sample-web/js/sharedState as SharedState">
```

There are console errors related to those dependencies
![Selection_077](https://user-images.githubusercontent.com/35235266/129817448-8e3d710d-ee9d-4e24-83ec-083e3905841c.png)

And the jsp portion is not displayed properly after publishing the portlet
![Selection_078](https://user-images.githubusercontent.com/35235266/129817444-8b16c5bb-357a-4df9-bf58-985764307f04.png)

Reply:
Hi @izaera , any idea on how to troubleshoot this? Is there a syntax or step that I may be missing to generalize the version dependency?

Reply:
Hi John! We've discussing this issue (with another module) here: https://liferay.slack.com/archives/C3JBR21HA/p1628698250243600

TL;DR: there is a PR that should fix it.

Reply:
Thanks!

Note: depends on PR https://github.com/liferay-frontend/liferay-portal/pull/1367#issuecomment-901456661

Reply:
It should work without the version numbers. `<aui:script require>` has support for that, as long as you declare the required packages in your project's `package.json` and use the bundler to build it...

Reply:
Look for `<aui:script require` inside portal's sourcetree and you'll see that nobody specifies the version number.